core/server/data/schema/schema.js

```diff
@@ -215,5 +215,21 @@ module.exports = {
         created_by: {type: 'integer', nullable: false},
         updated_at: {type: 'dateTime', nullable: true},
         updated_by: {type: 'integer', nullable: true}
+    },
+    invites: {
+        id: {type: 'increments', nullable: false, primary: true},
+        status: {type: 'string', maxlength: 150, nullable: false, defaultTo: 'pending', validations: {isIn: [['pending', 'sent']]}},
+        token: {type: 'string', maxlength: 191, nullable: false, unique: true},
+        email: {type: 'string', maxlength: 191, nullable: false, unique: true, validations: {isEmail: true}},
+        expires: {type: 'bigInteger', nullable: false},
+        created_at: {type: 'dateTime', nullable: false},
+        created_by: {type: 'integer', nullable: false},
+        updated_at: {type: 'dateTime', nullable: true},
+        updated_by: {type: 'integer', nullable: true}
+    },
+    invites_roles: {
+        id: {type: 'increments', nullable: false, primary: true},
+        role_id: {type: 'integer', nullable: false},
+        invite_id: {type: 'integer', nullable: false}
```

Comment:
thought about it as well and i agree, but changing this would mean we need to change client and server PR almost from stretch.

i think for each resource (permission, user, invites) we always have only one role relation or?
what if we add a sub task to #7432 reconsidering this approach?